fastify.js

```diff
@@ -408,9 +410,13 @@ function build (options) {
     }
 
     validateBodyLimitOption(opts.jsonBodyLimit)
-    var jsonBodyLimit = opts.jsonBodyLimit || _fastify._jsonBodyLimit
 
-    _fastify.after((notHandledErr, done) => {
+    _fastify.after(afterRouteAdded)
+
+    function afterRouteAdded (notHandledErr, done) {
+      onRouteHooks.forEach(h => h.call(_fastify, opts))
```

Comment:
Is there some reasons to use this short form instead of the longer for-loop form?

Personally I prefer
```
for (var i in onRouteHooks) {
  onRouteHooks[i].call(_fastify, opts)
}
```
Only for uniformity.

Thoughts?

Reply:
Do you mean 

```js
      for (var h of onRouteHooks) {
        h.call(_fastify, opts)
      }
```